lib/dor/text_extraction/ocr.rb

```diff
@@ -31,15 +31,15 @@ def required?
         workflow_context['runOCR'] || false
       end
 
-      private
-
       # return a list of filenames that should be OCR'd
       # iterate over all files in cocina_object.structural.contains, looking at mimetypes
       # return a list of filenames that are correct mimetype
       def filenames_to_ocr
         ocr_files.map(&:filename)
       end
 
+      private
```

Comment:
I needed filenames_to_ocr to be public so that the fetch_files step would know what files to get from Preservation.